cfme/web_ui/menu.py

```diff
@@ -127,6 +127,7 @@ def get_current_menu_state(self):
         unlimited levels of menu to be used, however it is unlikely that more than 3 will ever be
         used.
         """
+        self.initialize()
```

Comment:
@psav Thoughts here?

According to the doc for Menu, Initialize was called in force_navigate, but since we're no longer using force_navigate Menu needs to be initialized before `Menu.CURRENT_TOP_MENU` is set.

Should possibly initialize in __init__ now?